README.md

```diff
@@ -11,20 +12,47 @@ You can use this action to build images from an [Okteto Manifest](https://www.ok
 
 ## Inputs
 
+### `tag`
+
+Name and optionally a tag in the `name:tag` format for the build. When `file` points to a `Dockerfile`, in order to push the image to a registry, a `tag` is required. Otherwise, the build would be done but no image will be pushed to the registry.
+
 ### `file`
 
-The path to the Okteto Manifest. Default `"okteto.yml"`.
-Name of the Dockerfile. Default `"Dockerfile"`.
+The relative path to the Okteto Manifest.
+
+> You can also use this input to point to a Dockerfile. In this mode, okteto build will ignore your Okteto manifest, and directly build the image defined in the Dockerfile. Use this to build images that are not defined on your Okteto manifest.
+
+### `path`
+
+Service from the Okteto Manifest to build. You can select the service to build providing the `path`, otherwise all images at the Okteto Manifest build definition would be build.
+
+When repository does not have an Okteto Manifest or `Dockerfile` is provided at `file`, `path` is the execution path of the action. .
+
+### `buildargs`
 
-### `global`
```

Comment:
If we remove `global` we may break existing workflows, should we release this as a breaking change?

Reply:
global param is deprecated, the CLI does not use it. i think what should be done is to file an issue to remove this from the options available at CLI.